src/data/set/finite.lean

```diff
@@ -756,6 +758,15 @@ lemma to_finset_union {α : Type*} [decidable_eq α] (s t : set α) [fintype (s
   [fintype s] [fintype t] : (s ∪ t).to_finset = s.to_finset ∪ t.to_finset :=
 by ext; simp
 
+instance fintype_sdiff  {α : Type*} [decidable_eq α]
```

Comment:
Can we drop the instance name?

Reply:
I like giving names to instances if there's a chance you might ever want to use them explicitly. Instances are definitions after all. (This file gives similar names to other instances, too.)